### PR TITLE
Add history folder support

### PR DESCRIPTION
**Pull request by vbragin**

### CODE REVIEW

File: allure-teamcity-plugin-agent/src/main/java/ru/yandex/qatools/allure/AllureBuildServiceAdapter.java [hunk 1]
```diff
@@ -124,11 +124,12 @@ private void clearReport() throws IOException {
      */
     private void copyHistory() {
 
-        Path source = Paths.get(reportDirectory.toAbsolutePath().toString() + "/data/history.json");
+        Path source = Paths.get(reportDirectory.toAbsolutePath().toString() + "/history");
```
Comment (eroshenkoam) [inline allure-teamcity-plugin-agent/src/main/java/ru/yandex/qatools/allure/AllureBuildServiceAdapter.java]: Paths.get(reportDirectory.toAbsolutePath()).resolve("history")

File: allure-teamcity-plugin-agent/src/main/java/ru/yandex/qatools/allure/AllureBuildServiceAdapter.java [hunk 2]
```diff
@@ -123,12 +123,11 @@ private void clearReport() throws IOException {
      * Write the history file to results directory.
      */
     private void copyHistory() {
-
-        Path source = Paths.get(reportDirectory.toAbsolutePath().toString() + "/data/history.json");
+        Path source = reportDirectory.resolve("history");
         if (Files.exists(source)) {
```
Comment (baev) [inline allure-teamcity-plugin-agent/src/main/java/ru/yandex/qatools/allure/AllureBuildServiceAdapter.java]: add `isDirectory` check

**Comment by vbragin** [comment]
There is a problem with Report url.
Allure generate links in view like:
http://localhost:8111/repository/download/AllureTest_Junit/28:id/index.html/#xUnit/7aba711bd2e23944
but 
http://localhost:8111/repository/download/AllureTest_Junit/28:id/index.html#xUnit/7aba711bd2e23944
required.
The slash between index.html #xUnit is excess.

**Comment by eroshenkoam** [comment]
@baev ping

**Pull request merged**